charts/opentelemetry-demo/values.yaml

```
@@ -63,6 +77,7 @@ components:
     enabled: true
     useDefault:
       env: true
+    imageOverride: {}
```

We can call it `image` and use yaml anchors here
```suggestion
    image: *default-image
```